src/components/vm/snapshots/vmSnapshotsCreateModal.jsx

```diff
@@ -28,20 +28,25 @@ import { TextInput } from "@patternfly/react-core/dist/esm/components/TextInput"
 import { FormHelper } from 'cockpit-components-form-helper.jsx';
 import { DialogsContext } from 'dialogs.jsx';
 import { ModalError } from "cockpit-components-inline-notification.jsx";
+import { FileAutoComplete } from "cockpit-components-file-autocomplete.jsx";
 import { snapshotCreate, snapshotGetAll } from "../../../libvirtApi/snapshot.js";
+import { getSortedBootOrderDevices, LIBVIRT_SYSTEM_CONNECTION } from "../../../helpers.js";
 
 const _ = cockpit.gettext;
 
+let current_user = null;
+cockpit.user().then(user => { current_user = user });
+
 const NameRow = ({ onValueChanged, name, validationError }) => {
     return (
         <FormGroup
             label={_("Name")}
             fieldId="snapshot-create-dialog-name">
             <TextInput value={name}
-                validated={validationError.name ? "error" : "default"}
+                validated={validationError ? "error" : "default"}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1280-20240129-101752-d3606ddd-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)